<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>CrystalDialog</class>
 <widget class="QDialog" name="CrystalDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Input Crystal</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>460</y>
     <width>571</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>90</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title" >
    <string>Lattice</string>
   </property>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>a :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>b :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>c :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>beta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>alpha </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LatticeA" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>50</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LatticeB" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>80</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LatticeC" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>110</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LatticeAlpha" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>50</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LatticeBeta" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>80</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LatticeGamma" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>110</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>110</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>gamma</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>260</y>
     <width>571</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title" >
    <string>Unit Cell</string>
   </property>
   <widget class="QTableWidget" name="AtomsTable" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>551</width>
      <height>151</height>
     </rect>
    </property>
    <property name="autoFillBackground" >
     <bool>true</bool>
    </property>
    <property name="showGrid" >
     <bool>true</bool>
    </property>
    <property name="gridStyle" >
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="rowCount" >
     <number>2</number>
    </property>
    <property name="columnCount" >
     <number>4</number>
    </property>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>0</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title" >
    <string>Spacegroup</string>
   </property>
   <widget class="QRadioButton" name="CubicButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Cubic</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="TetragonalButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Tetragonal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>180</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Spacegroup</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MonoclinicButton" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>60</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Monoclinic</string>
    </property>
    <property name="checked" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="HexagonalButton" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>30</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Hexagonal</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="TriclinicButton" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>90</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Triclinic</string>
    </property>
    <property name="checked" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="TrigonalButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>120</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Trigonal</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="AllSpaceGroupsButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>List All Space Groups</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="OrthorhombicButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Orthorombic</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SpacegroupBox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>210</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="editable" >
     <bool>true</bool>
    </property>
    <property name="currentIndex" >
     <number>-1</number>
    </property>
    <property name="maxVisibleItems" >
     <number>15</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>0</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Information</string>
   </property>
   <widget class="QTextEdit" name="DescriptionTextEdit" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>49</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>CrystalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CrystalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
